lib/utils/config/definitions.js

```diff
@@ -1782,7 +1782,7 @@ define('scope', {
   `,
   flatten (key, obj, flatOptions) {
     const value = obj[key]
-    flatOptions.projectScope = value && !/^@/.test(value) ? `@${value}` : value
+    flatOptions.scope = value && !/^@/.test(value) ? `@${value}` : value
```

Comment:
This still need to be `projectScope` because that's where `npm-registry-fetch` uses this to set appropriate headers in its requests

https://github.com/npm/npm-registry-fetch/blob/8954f61d8d703e5eb7f3d93c9b40488f8b1b62ac/index.js#L202-L203